contrib.json

```diff
@@ -372,5 +379,53 @@
         [ "git branch -D {{pr.user.login}}-{{pr.head.ref}}",                   "Delete the local branch used for merging" ]
       ]
     }
+  },
+
+  "stage": {
+    "patch": {
+      "release_type": "patch",
+      "description": "Get the repo ready for releasing a patch",
+      "steps": [
+        { "include": "update stable" },
+        { "include": "update patch" },
+        [ "git merge stable", "Merge stable into patch to stage patch release" ],
+        [ "git push upstream path", "Push upstream to trigger patch release" ],
+        [ "git push origin path", "Update origin" ]
+      ]
+    }
+  },
+  "codeship": {
+    "run": {
+      "steps": [
```

Comment:
I think that totally makes sense. If I were to clone videojs directly and have `origin` set up as `videojs/video.js`, then, it will work. I thought I made a comment about the `codeship` name here but it got lost, apparently. But basically, I called it `codeship` because that's where it'll be run. It's possible that just moving it to be a sub section of `release` will work. But then naming is weirder.